Dockerfile.konflux

```diff
@@ -0,0 +1,64 @@
+ARG DEV_IMAGE=registry.redhat.io/ubi9/go-toolset@sha256:fd41c001abc243076cc28b63c409ae6d9cbcad401c8124fb67d20fe57a2aa63a
+
+FROM registry.redhat.io/ubi9/go-toolset@sha256:fd41c001abc243076cc28b63c409ae6d9cbcad401c8124fb67d20fe57a2aa63a AS build
+# https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope
+# don't provide "default" values (e.g. 'ARG TARGETARCH=amd64') for non-buildx environments,
+# see https://github.com/docker/buildx/issues/510
+ARG TARGETOS=linux
+ARG TARGETARCH=amd64
+
+LABEL image="build"
+
+# Copy the go sources
+COPY main.go main.go
+COPY apis/ apis/
+COPY controllers/ controllers/
+COPY generated/ generated/
+COPY pkg/ pkg/
+COPY version /etc/modelmesh-version
+
+USER root
+
+COPY go.mod go.mod
+COPY go.sum go.sum
+RUN go mod download
+
+# Build using native go compiler from BUILDPLATFORM but compiled output for TARGETPLATFORM
+RUN GOOS=${TARGETOS:-linux} \
+    GOARCH=${TARGETARCH:-amd64} \
+    CGO_ENABLED=0 \
+    GO111MODULE=on \
+    go build -a -o /workspace/manager main.go
+
+###############################################################################
+# Stage 2: Copy build assets to create the smallest final runtime image
+###############################################################################
+FROM registry.redhat.io/ubi8/ubi-minimal@sha256:7583ca0ea52001562bd81a961da3f75222209e6192e4e413ee226cff97dbd48c AS runtime
+
+ARG USER=2000
+ARG IMAGE_VERSION
+ARG COMMIT_SHA
+
+LABEL com.redhat.component="odh-modelmesh-serving-controller-container" \
+      name="managed-open-data-hub/odh-modelmesh-serving-controller-rhel8" \
+      description="Manages lifecycle of ModelMesh Serving Custom Resources and associated Kubernetes resources" \
+      summary="odh-modelmesh-serving-controller" \
+      maintainer="['managed-open-data-hub@redhat.com']" \
+      io.openshift.expose-services="" \
+      io.k8s.display-name="modelmesh-serving-controller" \
+      io.k8s.description="modelmesh-serving-controller" \
+      com.redhat.license_terms="https://www.redhat.com/licenses/Red_Hat_Standard_EULA_20191108.pdf"
+
+
+## Install additional packages
+RUN microdnf install -y shadow-utils &&\
```

Comment:
why are you adding this if no user is being added?

Reply:
I have not added it, referred from the midstream dockerfile

Reply:
we are not adding users, it can be removed..